openmetadata-service/src/main/java/org/openmetadata/csv/CsvUtil.java

```diff
@@ -205,4 +272,96 @@ private static String quoteCsvField(String str) {
     }
     return str;
   }
+
+  public static List<String> addExtension(List<String> csvRecord, Object extension) {
+    if (extension == null) {
+      csvRecord.add(null);
+      return csvRecord;
+    }
+
+    ObjectMapper objectMapper = new ObjectMapper();
+    Map<String, Object> extensionMap = objectMapper.convertValue(extension, Map.class);
+
+    String extensionString =
+        extensionMap.entrySet().stream()
+            .map(
+                entry -> {
+                  String key = entry.getKey();
+                  Object value = entry.getValue();
+                  return CsvUtil.quoteCsvField(key + ENTITY_TYPE_SEPARATOR + formatValue(value));
+                })
+            .collect(Collectors.joining(FIELD_SEPARATOR));
+
+    csvRecord.add(extensionString);
+    return csvRecord;
+  }
+
+  private static String formatValue(Object value) {
+    // Handle Map (e.g., entity reference or date interval)
```

Comment:
no need for these comments, are these getting auto generated?

Reply:
yes the comments part have been auto-generated, removed the additional comments